test/async-hooks/test-get-connections.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const assert = require('assert');
+const net = require('net');
+const server = net.createServer();
+
+try {
+    server.getConnections(() => {
+        assert.ok(true, "No error thrown")
+    });
+}
+catch (e) {
+    assert.ok(false, "getConnections threw an error");
+}
```

Comment:
OK, would you prefer a comment to that effect then? I was trying to write the test showing the context of what happened before the change.